### PR TITLE
[IMP] mail, forum, utm: cleanup useless rules

### PR DESCRIPTION
**Pull request by mart-e**
As mentionned in 65530dfd6a04, the composer rights was only needed for forum Remove unused rules and add sudo where needed.

While being there let us remove useless rules for UTM records.

Task-3800555

### CODE REVIEW

**Comment by mart-e** [comment]
robodoo r+

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
Ah but you did self r+ @mart-e . You are that kind of guy.

**Comment by mart-e** [comment]
I am also the kind of guy who self
@robodoo r-
does it mean I am robodoo r0? robodoo moyen bof?

**Comment by tde-banana-odoo** [comment]
Probably robodoo moyen-bof+ . Not shining robodoo.

File: addons/test_mail/tests/test_mail_composer.py [hunk 1]
```diff
@@ -1189,27 +1189,15 @@ def test_mail_composer_rights_portal(self):
...
-                'default_composition_mode': 'comment',
-                'default_parent_id': self.test_record.message_ids.ids[0],
-            }).create({
-                'subject': 'Subject',
-                'body': '<p>Body text 2</p>'
-            })._action_send_mail()
-
-            self.assertEqual(self.test_record.message_ids[0].body, '<p>Body text 2</p>')
-            self.assertEqual(self.test_record.message_ids[0].author_id, portal_user.partner_id)
+            with self.assertRaises(AccessError):
```
Comment (tde-banana-odoo) [inline addons/test_mail/tests/test_mail_composer.py:1192]: Could you link Task-3800555 in commit message ? Just for tracking :)

File: addons/website_forum/models/forum_post.py [hunk 2]
```diff
@@ -336,7 +336,7 @@ def create(self, vals_list):
             # add karma for posting new questions
             if not post.parent_id and post.state == 'active':
                 post.create_uid.sudo()._add_karma(post.forum_id.karma_gen_question_new, post, _('Ask a new question'))
-        posts.post_notification()
+        posts.sudo().post_notification()
```
Comment (tde-banana-odoo) [inline addons/website_forum/models/forum_post.py:339]: Can't we take the opportunity to use a better name, like ``_notify_state_update`` ?

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**